helpers/javascript/lib/updater.js

```diff
@@ -15,7 +15,7 @@ const fs = require("fs");
 const path = require("path");
 const { Add } = require("yarn/lib/cli/commands/add");
 const Config = require("yarn/lib/config").default;
-const { NoopReporter } = require("yarn/lib/reporters");
+const { EventReporter } = require("yarn/lib/reporters");
```

Comment:
NoopReporter means we don't get proper exception messages, EventReporter also effectively Noops, but fixes that